src/main/java/com/iotiq/application/exception/UnremovableSkillException.java

```diff
@@ -0,0 +1,12 @@
+package com.iotiq.application.exception;
+
+import com.iotiq.commons.exceptions.ApplicationException;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+public class UnremovableSkillException extends ApplicationException {
+    public UnremovableSkillException() {
+        super(HttpStatus.NOT_ACCEPTABLE, "unremovableSkill", List.of(), new Object[]{});
```

Comment:
Does not acceptable fit this exception? I'm not sure. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406 It seems like it's returned when the response type is not supported. 
![image](https://github.com/user-attachments/assets/b1346147-e368-418e-9a0d-78bacb901d39)